browsers/linux.py

```diff
@@ -44,7 +44,7 @@ def browsers() -> Iterator[Browser]:  # type: ignore[return]
                     executable_path = entry.getExec()
                     if executable_path.lower().endswith(" %u"):
                         executable_path = executable_path[:-3].strip()
-                    version = subprocess.getoutput(f"{executable_path} --version 2>&1").strip()
+                    version = subprocess.check_output([executable_path, "--version"]).decode().strip()
```

Comment:
I still have to test this later because `--version` can sometimes include error messages on some platforms/browsers, hence the `2>&1`

Reply:
For consistency I have added the `2>&1` argument back in.